prow/jobs/kyma-project/compass-manager/golang-linter.yaml

```diff
@@ -0,0 +1,37 @@
+presubmits: # runs on PRs
+  kyma-project/compass-manager:
+    - name: pre-main-compass-manager-lint
+      annotations:
+        pipeline.test: "fast-integration"
+        pipeline.trigger: "pr-submit"
+        pipeline.type: "integration"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-compass-manager-lint"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+      optional: true
+      run_if_changed: '^(go.mod|go.sum)$|^(.*\.go$|.golangcilint.yaml)'
+      skip_report: false
+      decorate: true
+      path_alias: github.com/kyma-project/compass-manager
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^main$
+      spec:
+        containers:
+          - image: "europe-docker.pkg.dev/kyma-project/dev/test-infra/golangci-lint:PR-7337"
```

Comment:
Please don't use PR image here.

Reply:
In the [Artifactory Registry](https://console.cloud.google.com/artifacts/docker/kyma-project/europe/dev/test-infra%2Fgolangci-lint?authuser=1), I can't find images that aren't PR's images. Could you please provide where I can find the right images?